builder/builder.go

```diff
@@ -1640,6 +1647,64 @@ func (b *Builder) BuildImage(format string, template string) error {
 	return imagecmd.Run()
 }
 
+type dnfRepoConf struct {
+	RepoName, RepoURL string
+}
+
+const dnfConfRepoTemplate = `
```

Comment:
Really minor issue, but perhaps add a blank line at the start of this template so that the INI sections are more clearly separated?

Reply:
It adds a newline at the beginning and end of the template already. Therefore if you keep adding repos there will always be a blank line between the sections. The other base template also adds a trailing newline. I'd rather not have two newlines between the sections.

Reply:
Okay, I agree.